target/linux/ipq40xx/image/Makefile

```diff
@@ -340,6 +340,35 @@ define Device/netgear_ex6100v2
 endef
 TARGET_DEVICES += netgear_ex6100v2
 
+define Device/p2w_r619ac
```

Comment:
So, what's the different between this device and the following one? Why do I not learn about two devices being added in the commit message?